che-plugin-flux-live-edit.iml

```diff
@@ -1,75 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
removed here and added in https://github.com/sunix/che-plugin-flux-live-edit/pull/5/commits/fe793eaaead329b141129d901cc2d9e85a2c0b2b
This has nothing to do with what is describe "New class has been added ..."